src/main/java/frc/robot/Other/Explanations.java

```diff
@@ -0,0 +1,92 @@
+package frc.robot.Other;
+
+import frc.robot.subsystems.*;
+import frc.robot.Constants;
+import frc.robot.Robot;
+import frc.robot.RobotContainer;
+import frc.robot.commands.*;
+
+/**
+ * Here, Explanations of How everything works are held.
+ *
+ * <p>When Within The class:
+ *
+ * <p>{@code - Hover over the Green Part to Get a basic explanation}
+ *
+ * <p>{@code - Hover over the Yellow Part for a More Detailed explanation}
+ *
+ * <p>{@code - Hover over The Blue Part to find what your getting an Explanation of}
+ *
+ * <p>EX: int {@link #DriveTrainExplanation} (Blue) = {@link DriveTrain} (Green) {@link
+ * DriveTrain#Explanation} (Yellow)
+ *
+ * <p>Sub Explanations are down Below:
+ *
+ * <ul>
+ *   <li>Init Sendable:
+ *       <ul>
+ *         <li>Sendable objects are things that can be sent to the smart dashboard (WHEN TALKING
+ *             ABOUT FRC!!!!).
+ *         <li>Initializing a sendable object sends that object to the smart dashboard.
+ *         <li>To send other things to the smart dashboard, Use the sendable builder (the thing that
+ *             sends stuff)
+ *         <li>EX: {@link Hanger#initSendable} and {@link DriveTrain#initSendable}
+ *         <li>In order to send variables, you must do {@code builder.addDoubleProperty} (Instead of
+ *             Double do variable type of choice)
+ *       </ul>
+ */
+public class Explanations {
+
+  public abstract class SubsystemExplanations {
```

Comment:
Why is the public abstract class?

Reply:
Dont wont to make an object of it.

Reply:
Why make an object out of it at all? Why is this a class?

Reply:
want*

Reply:
Right, why is this a class at all?

Reply:
I wanted to make it clear for younger students where all the explanataions would be

Reply:
When people want explanation for things I would like for everything to be in the same place

Reply:
If you look in Hanger Module The explanation is like a good 200 words that I dont want to be shown when hovering over HangerModule.java

Reply:
Break the comments into their individual functions then.